app/commands/assets/soft_delete_relations.rb

```diff
@@ -0,0 +1,18 @@
+module Assets
```

Comment:
It feels preemptive to namespace commands to me (it only adds noise at this point) since it's our first command. It might be cleaner to do something like https://github.com/nslocum/design-patterns-in-ruby/blob/master/commands/create_file.rb and call this `SoftDeleteAssetWithRelations`

Reply:
My reasoning is that it will change soon and we can add things like:
```
app
   \_ commands
     \_ add_waveform.rb
     \_ ensure_unique_permalink.rb
     \_ notify_followers.rb
```